libs/langchain/langchain/vectorstores/mongodb_atlas.py

```diff
@@ -254,33 +339,33 @@ def similarity_search(
     def max_marginal_relevance_search(
         self,
         query: str,
-        k: int = 4,
+        limit: int = 4,
```

Comment:
This seems like an unrelated api change that would be better in a separate PR.

Reply:
Good catch, thought this was a non-public API for some reason.